frb_codegen/Cargo.toml

```diff
@@ -1,10 +1,12 @@
+cargo-features = ["workspace-inheritance"]
```

Comment:
nit: so... why do we need this feature? (guess it is because clippy?)

Reply:
older versions of cargo require this line to be present in order to use workspace dependency inheritance. (however, is now unneeded)

By adding this line alone, the MSRV dropped quite a bit.